Functions/PollEmailProcessing.cs

```diff
@@ -0,0 +1,60 @@
+using System.Text.RegularExpressions;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace Functions
+{
+    public class PollEmailProcessing
+    {
+        private readonly ILogger _logger;
+
+        public PollEmailProcessing(ILoggerFactory loggerFactory)
+        {
+            _logger = loggerFactory.CreateLogger<PollEmailProcessing>();
+        }
+
+        [Function("PollEmailProcessing")]
+        public async Task RunAysnc([TimerTrigger("* * * * *")] MyInfo myTimer)
+        {
+            var url = Environment.GetEnvironmentVariable("PollEmailProcessingUrl", EnvironmentVariableTarget.Process);
+            url ??= "https://find-a-tuition-partner-qa.london.cloudapps.digital/admin/process-emails";
```

Comment:
Do we need to fall back to the QA URL when we have not specified an URL via the environment variable? Or should we throw an exception saying the URL is not specified via the environment variable?

Reply:
Agreed, exception will be better here.